src/tarkov-data-manager/data/changed_quests.json

```diff
@@ -487,5 +487,47 @@
                 }
             ]
         }
+    },
+    "5ae4493486f7744efa289417": {
+        "name": "Database - Part 1",
+        "propertiesChanged": {
+            "taskRequirements": [
+                {
+                    "task": "5ae448bf86f7744d733e55ee",
+                    "name": "Make ULTRA Great Again",
+                    "status": [
+                        "complete"
+                    ]
+                }
+            ]
+        }
+    },
+    "5ae448f286f77448d73c0131": {
+        "name": "The Blood of War - Part 1",
+        "propertiesChanged": {
+            "taskRequirements": [
+                {
+                    "task": "5ae448bf86f7744d733e55ee",
+                    "name": "Make ULTRA Great Again",
+                    "status": [
+                        "complete"
+                    ]
+                }
+            ]
+        }
+    },
+    "6086c852c945025d41566124": {
+        "name": "Revision - Reserve",
```

Comment:
Also, **Revision - Lighthouse** and **Revision - Streets of Tarkov**

Reply:
Actualy, from the wiki it seems like that the **Revision - Lighthouse** is given after you accept **Revision - Reserve**, and **Revision - Streets of Tarkov** requires both **Humanitarian Supplies** and **Friend from the West - Part 2**.

Reply:
Is Revision - Lighthouse available immediately after completing Humanitarian Supplies, or is it available once you start Revision - Reserve? The Wiki says the latter (which could be old info).

Reply:
I would trust wiki on this as it's new info added/updated after the wipe. I will complete **Humanitarian Supplies** today and report what happens to these two quests ;)

Reply:
So, finishing **Humanitarian Supplies** has unlocked me **Revision - Reserve**.
As soon as I accepted **Revision - Reserve** both **Revision - Lighthouse** and **Revision - Streets of Tarkov** become available (I already completed _Friend from the West - Part 2_ so can't verify that that's a correct prerequisite, let's trust wiki :) )

Reply:
Sounds good!